# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/tlp/hotfix-install.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- TLP-1.7.0/Makefile.vanilla	2025-01-03 19:25:17.440001415 -0500
+++ TLP-1.7.0/Makefile	2025-01-03 19:31:26.776672271 -0500
@@ -150,9 +150,11 @@
 	install -m 755 tlp-run-on $(_BIN)/run-on-ac
 	ln -sf run-on-ac $(_BIN)/run-on-bat
 	install -m 755 tlp-stat $(_BIN)/
-	install -D -m 755 -t $(_TLIB)/func.d func.d/*
+	mkdir -p $(_TLIB)/func.d
+	install -D -m 755 func.d/* -t $(_TLIB)/func.d
 	install -m 755 tlp-func-base $(_TLIB)/
-	install -D -m 755 -t $(_TLIB)/bat.d bat.d/*
+	mkdir -p $(_TLIB)/bat.d
+	install -D -m 755 bat.d/* -t $(_TLIB)/bat.d
 	install -m 755 tlp-pcilist $(_TLIB)/
 	install -m 755 tlp-readconfs $(_TLIB)/
 	install -m 755 tlp-usblist $(_TLIB)/
